tests/cypress/e2e/unit_tests/p1_fleet.spec.ts

```diff
@@ -120,14 +122,15 @@ describe('Test resource behavior after deleting GitRepo using keepResources opti
   keepResourceData.forEach(
     ({ qase_id, keepResources, test_explanation}) => {
       qase(qase_id,
-        it(`Test ${test_explanation}`, { tags: `@fleet-${qase_id}` }, () => {
+        it(`Fleet-${qase_id}: Test ${test_explanation}`, { tags: `@fleet-${qase_id}` }, () => {
           cy.fleetNamespaceToggle('fleet-local')
           cy.addFleetGitRepo({ repoName, repoUrl, branch, path, keepResources });
           cy.clickButton('Create');
           cy.checkGitRepoStatus(repoName, '1 / 1', '1 / 1');
+          cy.checkApplicationStatus(appName);
           cy.deleteAllFleetRepos();
           if (keepResources === 'yes') {
-            cy.checkApplicationStatus(appNamespace, appName);
+            cy.checkApplicationStatus(appName);
```

Comment:
I understand the removal of `appNamespace` here was avoid calling `nameSpaceMenuToggle` function that is causing the error because it finds 2 locators. However, even when this may be solved here, the test keeps failing on the next function (`cy.deleteApplicationDeployment`) because it is called there as well.

In order to avoid this error, one solution could be changing locator on function `nameSpaceMenuToggle` from `cy.get('.icon.icon-chevron-up').click({ force: true });` to `cy.get('div.ns-dropdown.ns-open > i.icon.icon-chevron-up').click({ force: true });`

Reply:
I will check that out, locally it worked for me, as it removed the application after `keepResources: true` test is done. Still I will double check it and also, test with locator suggestion you gave.

Reply:
I can remove the `appNamespace` dependency from `cy.deleteApplicationDeployment`, as when navigate to `Workloads --> Deployment` it shows single deployment by having User Selected Namespace.

What is your opinion here ?